pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -269,6 +269,9 @@ type PoolSpec struct {
 
 	// The erasure code settings
 	ErasureCoded ErasureCodedSpec `json:"erasureCoded"`
+
+	// PoolProperties is a list of properties to enable on a given pool
+	PoolProperties map[string]string `json:"poolProperties,omitempty"`
```

Comment:
No need for the "pool" prefix since we're already in the PoolSpec
```suggestion
	Properties map[string]string `json:"properties,omitempty"`
```

Reply:
I'm not sure if that's going to work with the `openAPIV3Schema` which uses the `properties` keyword, that's why I used `poolProperties` in the first place. I can change it to `parameters`.